src/main/kotlin/com/desmonddavid/bots/digitforumbot/bots/NewPostsBot.kt

```diff
@@ -132,7 +137,36 @@ object NewPostsBot : TelegramLongPollingBot() {
                 }\n\n"
             message += entry
         }
-
+        if(checkIfNew(message) ==1){
```

Comment:
Move this condition to sendNewPostsMessage() method above (to line 117) and refactor as follows;

if(checkIfNew(sendMessage.text)) {
    // Do send message stuff.
}

Also, make checkIfNew() return boolean, refer comment below.